### DISCUSSION TITLE
How to delete a highlight on kindle 3(a non-touch device)?

### DISCUSSION THREAD

**Discussion by zhanglin2022** [koreader/koreader#12319] [Q&A] [answered]
My device is kindle 3(kindle keyboard) which is a non-touch device. 

How to delete a highlight after I open "TOP MENU-?-Bookmarks"? I tried to press the physical key "Del" on kindle 3, but it didn't work.

**Comment by jonnyl2** [comment]
Summoning @Commodore64user :)

**Comment by Commodore64user** [comment, marked as answer]
you have to do a long press on the specific highlight you want to delete (much like in a touch ereader). You MUST be running 2024.07 (or later) for this though

To do so, simply place the focus on the undesired highlight and press the following key combination: <kbd>shift</kbd> + <kbd>press</kbd>.

Note: long press is now functional throughout the whole application, so feel free to long press around to set defaults and all that fun stuff you could not do before.

BTW: if you update to the latest nightly, you have a new cool option to hide the virtual keyboard #12162, would be nice if you could try it out. And more importantly, report back if you see anything amiss.

**Reply by zhanglin2022** [reply]
Thanks, it worked.
